packages/flutter_tools/lib/src/build_system/targets/assets.dart

```diff
@@ -48,48 +48,57 @@ class AssetBehavior extends SourceBehavior {
   }
```

Comment:
In `AssetBehavior.outputs` the for loop doesn't touch entry.value, so I guess it could instead loop over assetBundle.entries.values instead. I notice that that's what `inputs` is doing, but `inputs` is also filtering on the type of the value. If `outputs` doesn't need to do that as well, then a comment there would be good.